packages/mui-base/src/SnackbarUnstyled/SnackbarUnstyled.test.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { createRenderer, createMount, describeConformanceUnstyled } from 'test/utils';
+import SnackbarUnstyled, { snackbarUnstyledClasses as classes } from '@mui/base/SnackbarUnstyled';
+
+describe('<SnackbarUnstyled />', () => {
```

Comment:
Let's replicate the tests we have in the Material UI's Snackbar component, so that we are sure that the features that were working before are still working for this component too. The main benefit is that we would be confident we are not creating any regressions compared to the version in Material UI.

Reply:
Sure. Added.